src/test/providers.tsx

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
-import { createStore, applyMiddleware } from 'redux';
+import { createStore, applyMiddleware, DeepPartial } from 'redux';
 import { ReactNode } from 'react';
 import {
+  State,
   reducer,
   NetworkManager,
   SubscriptionManager,
   ExternalCacheProvider,
   RestProvider,
 } from '../index';
 
+// Extension of the DeepPartial type defined by Redux which handles unknown
+type DeepPartialWithUnkown<T> = {
+  [K in keyof T]?: T[K] extends unknown
+    ? any
+    : (T[K] extends object ? DeepPartial<T[K]> : T[K])
```

Comment:
this should be calling DeepPartialWithUnkown else it only works on one level

Reply:
good point